docs/pages/guides/environment-variables.mdx

```diff
@@ -103,17 +103,17 @@ function Post() {
 
 This approach this plugin will work well with environment variables set in your shell process, but it won't automatically load **.env** files. We recommend using [direnv](https://direnv.net/) and **.envrc** to accomplish that.
 
-## Storing environment variables in .envrc or .env
+## Loading environment variables from a file
 
-If you want to automatically load environment variables from a file in your project rather than setting them in your shell profile or manually when running `npx expo start`, you may want to use a **.envrc** or **.env** file.
+If you want to automatically load environment variables from a file in your project rather than setting them in your shell profile or manually when running `npx expo start`, you may want to use an environment variable manager.
 
 ### direnv
 
-[direnv](https://direnv.net/) automatically loads and unloads environment variables in your shell depending on your current directory. When you `cd` into your project directory, it will load all of the variables as configured in **.envrc**. [Learn more](https://direnv.net/#getting-started).
+[direnv](https://direnv.net/) automatically loads and unloads environment variables in your shell depending on your current directory. When you `cd` into your project directory, it will load all of the variables as configured in **.envrc**. Due to its tight shell integration, we highly recommend direnv for the highest level of compatiblity with all Expo and EAS CLI tooling. [Learn more](https://direnv.net/#getting-started).
 
 ### dotenv
 
-[dotenv](https://github.com/motdotla/dotenv) is a library that you import in your project code to load the **.env** file. [Learn more](https://github.com/motdotla/dotenv).
+[dotenv](https://github.com/motdotla/dotenv) is a library that you import in your project code to load the **.env** file. Since it works at the JavaScript code level, dotenv may be affected by your Node environment or how you invoke it from your code, so be sure to check their documentation for the correct usage if you run into issues. [Learn more](https://github.com/motdotla/dotenv).
```

Comment:
```suggestion
[dotenv](https://github.com/motdotla/dotenv) is a library that you import in your project code to load the **.env** file. Since it works at the JavaScript code level, dotenv may be affected by your Node.js environment or how you invoke it from your code. Check the [documentation](https://github.com/motdotla/dotenv) for the correct usage if you run into issues.
```